1

```diff
@@ -0,0 +1,33 @@
+pick 7ec61c565 Add V1 version of Pipeline CRD
```

Comment:
I don't think this should be comitted

Reply:
I'm so very sorry. Didn't know how this get squeezed in by my vscode.